Update Networking intro and move Multiplayer Gem topics underneath (#2051)
o3de/o3de.org: content/docs/user-guide/networking/aznetworking/_index.md

---
linktitle: AzNetworking
title: AzNetworking Framework
description: Learn about the low-level networking stack in Open 3D Engine.
weight: 100
---

> `AzNetworking` is an OS-level network interface that provides simple, fast, and efficient networking by focusing on reduced code size and complexity, low latency on packet send and receive operations, and low message processing overhead.

Reviewer: ```suggestion
`AzNetworking` is a low-level network interface that provides simple, fast, and efficient networking by focusing on reduced code size and complexity, low latency on packet send and receive operations, and low message processing overhead.
```

## Section topics

| Topic | Description |
|---|---|

> | [Packet Structure](./packets) | Information on the packet structure used for TCP and UDP packets by `AzNetworking`, and how to manage fragmented UDP packets. |

Reviewer: ```suggestion
| [Packet Structure](./packets) | Information on the packet structure used for TCP and UDP packets in `AzNetworking`, and how to manage fragmented UDP packets. |
```
